assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -54,9 +57,16 @@ const Header = styled.h2`
   cursor: pointer;
 `;
 
-const Heading = styled(Headline)`
-  color: ${({ theme }) => theme.colors.fg.secondary};
-  line-height: 32px;
+const Heading = styled.span`
+  color: ${({ theme }) => theme.colors.fg.primary};
```

Comment:
This is actually `fg.primary` only when the panel is opened -- it used to be `fg.secondary` for all panels but changed to `fg.primary` for expanded panels. I'm also adjusting this [here](https://github.com/google/web-stories-wp/pull/6737/files#diff-a54a124ac5a73d9a8cecd8ccfb719d1cf8bff8738fa127b72f8779ad7c36bf3aR59).